test/provider_test.dart

```diff
@@ -72,5 +72,37 @@ void main() {
             });
       });
     });
+
+    group('getStorageAt', () {
+      test('returns the ERC20_name value for a ERC20 contract', () async {
+        final response = await provider.getStorageAt(
+          contractAddress: StarknetFieldElement.fromHex(
+              '0x0335c0d0c2b25730b7ed46e0fceed2a55d7743e300f393535c88470e5e15ae64'),
+          key: StarknetFieldElement.fromHex(
```

Comment:
You can use `getSelectorByName`, this should make the code simpler to understand.